### ISSUE TITLE
Ability to read byte range [Feature Request]

### ISSUE THREAD

**Issue by macdabby** [johanneslumpe/react-native-fs#153]
In some instances, uploading a file in chunks is more reliable - an uploaded file can be uploaded partially then continued later if the connection is lost. In order to do this, it would be great if a file stream could load just a segment of a file starting at a particular offset and loading a specific number of bytes.

**Comment by johanneslumpe** [comment]
@macdabby I agree that this would be a nice addition. Right now I cannot devote any time to this though. Feel like submitting a PR to add this functionality for both iOS and Android?

**Comment by macdabby** [comment]
Yeah, I was thinking about it :) I'm still very new to RN so it might take me a while to figure some of it out.

**Comment by macdabby** [comment]
Hey, what do you know, this is pretty easy :D I don't have my android environment setup yet, but I'll try to get that working next. Here is a PR for iOS using the same methods with additional params:
https://github.com/johanneslumpe/react-native-fs/pull/160

**Comment by faceyspacey** [comment]
+1
